stacks/spica/src/styles.scss

```diff
@@ -103,3 +106,7 @@ mat-form-field {
     background-color: #fafafa !important;
   }
 }
+
+.mat-list-base .mat-list-item {
+  font-size: 1rem;
+}
```

Comment:
We can not put this global style here. It should be in the component scope. If we leave this here, it will affect places that it not supposed to.

Reply:
It is an intended change. We actually changed the theme settings and override font-size:16px.

Reply:
Then could you do it by using the material's typography mixins?

https://material.angular.io/guide/typography

Reply:
Change requests accepted